src/renderer/components/UserFeedback/ReportBugModal.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  Box,
+  Typography,
+  Stack,
+  styled,
+  Modal,
+  TextField,
+  Snackbar,
+} from '@mui/material';
+import { useState } from 'react';
+import colors from 'renderer/colors';
+import {
+  PrimaryButton,
+  SecondaryButton,
+} from 'renderer/components/Blocks/Buttons';
+import ipc from '../../ipc';
+
+const STATUS_OK = 200;
+
+const { reportBug } = ipc;
+
+const CustomStack = styled(Stack)`
+  width: 100%;
+  height: 100%;
+  justify-content: space-between;
+`;
+
+const CustomModal = styled(Modal)`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const CustomModalInner = styled(Box)`
+  background: ${colors.grey[700]};
+  padding: 15px 30px 30px 30px;
+  border-radius: 5px;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  width: 400px;
+`;
+
+const CustomRowStack = styled(CustomStack)`
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+interface Props {
+  open: boolean;
+  onClose: () => void;
+}
+
+const ReportBugModal = ({ open, onClose }: Props) => {
+  const [bugTitle, setBugTitle] = useState('');
+  const [bugDescription, setBugDescription] = useState('');
+  const [showSnackbar, setShowSnackbar] = useState(false);
+  const [snackbarMesasge, setSnackbarMessage] = useState('');
+
+  const handleBugTitleInput = (value: string) => {
+    setBugTitle(value);
+  };
+
+  const handleBugDescInput = (value: string) => {
+    setBugDescription(value);
+  };
+
+  const onCloseBugModal = () => {
+    setBugTitle('');
+    setBugDescription('');
+    onClose();
+  };
+
+  const onSubmit = async () => {
+    const status = await reportBug(bugTitle, bugDescription);
+
+    setSnackbarMessage(
+      status === STATUS_OK
```

Comment:
I would recommend using the existing `AsyncState` primitive, that way we can handle loading as well. When testing, I was waiting a few seconds for the bug report to submit and didn't know if it had gone through or not. Also, it would be really easy to accidentally click the button multiple times and submit the same bug, so it should be greyed out when the report is loading.

Reply:
I've changed it so that the button disables once it gets pressed once. Also the snackbar message appears once the bug report has been submitted.

Reply:
👍🏻